onboarding/src/Components/hooks/Packages.js

```diff
@@ -78,7 +68,7 @@ export function singleCombo(packageId){
         fetchProps = {method: "GET", headers: {"Accept": "application/json", "Content-Type": "application/json", "X-CSRFToken": ""}};
 
     useEffect(() => {
```

Comment:
`ESLint` complains, but there is no reason for error - only `react` method in ordinary function.
`// eslint-disable-next-line react-hooks/rules-of-hooks` should be fine